hamilton/function_modifiers/dependencies.py

```diff
@@ -176,6 +176,9 @@ def _validate_group_params(
                     f"Dependency: {dependency} is not a valid dependency type for group(), must be "
                     f"a LiteralDependency or UpstreamDependency."
                 )
+    raise InvalidDecoratorException(
```

Comment:
This will always raise, no? Will want to add `return` statements above or something to get it to work.

Reply:
Oops! Thanks 👍